ydb/docs/en/core/reference/ydbops/install.md

```diff
@@ -2,30 +2,89 @@
 
 {% include [warning.md](_includes/warning.md) %}
 
-Currently, only building from source is available as an installation option. The pre-compiled executables will be available later.
-
 ## Building from source
 
-1. [Install Go](https://go.dev/doc/install). The minimal required Go version is 1.21.
-
-2. Clone the `ydbops` repository from GitHub:
+1. Clone the `ydbops` repository from GitHub:
 
     ```bash
     git clone https://github.com/ydb-platform/ydbops.git
     ```
 
-3. Invoke `go build` in the repository root folder:
+2. There are two ways to build `ydbops`:
+    2.1. [Directly with Go](#go)
+    2.2. [Inside a Docker container](#docker)
 
-    ```bash
-    go build
-    ```
+We are using official docker image for Golang v1.22. It gurantiees successful build. Description Docker container works in the Dockerfile in the repository. The assembly in Docker also performs additional manipulations: linter checks, the version for the ydbops assembly is substituted so that it is registered in the ydbops executable file.
 
-    The `ydbops` executable will be available in the repository root folder.
+### Building directly with Go {#go}
 
-## Download the binary from releases page
+#### Prerequsites
+
+[Install Go](https://go.dev/doc/install). The recommended version is 1.22
+
+
+#### Compiling
+
+Invoke `go build` in the repository root folder:
+
+```bash
+go build
+```
+
+The `ydbops` executable will be available in the repository root folder.
+
+#### Installing
+
+You can copy executable file manually or use `make`
+
+```bash
+make install INSTALL_DIR=install_folder BUILD_DIR=.
+```
+
+### Inside a Docker container {#docker}
+
+#### Prerequsites
 
-{% note warning %}
+- make
+- [Install docker engine](https://docs.docker.com/engine/install/)
 
-This option has yet to be made available.
+#### Compiling
+
+Invoke this command in the repository root folder:
+
+```bash
+make build-in-docker
+```
+
+The `ydbops` executables will be available in the `bin` folder. Binary files are generated for Linux and MacOS (arm64, amd64).
+
+| Binary name | Platform
+|-|-|
+| ydbops | Linux(amd64) |
+| ydbops_darwin_amd64 | MacOS(amd64) |
+| ydbops_darwin_arm64 | MacOS(arm64) |
+
+
+#### Installing
+
+To install the binary file, execute command `make`.
+
+Optional parameters:
+
+- `INSTALL_DIR`: The folder, to which the executable file will be installed. Default value: `~/ydb/bin`.
+
+- `BUILD_DIR`: The folder that contains the generated executable file. Use this parameter if you created the executable file manually. For example, use `BUILD_DIR=.` if the executable file is in the current working folder.
+
+```bash
+make install [INSTALL_DIR=<path_to_install_folder>] [BUILD_DIR=<path_to_build_folder>]
+```
+
+Sample command to install into `install_folder` from the current folder:
+
+```bash
+make install INSTALL_DIR=install_folder BUILD_DIR=.
+```
+
+## Download the binary from releases page
```

Comment:
Let's swap the order between downloading and self-building. People are unlikely to need to compile themselves and might waste time if they miss this download link at the bottom.